Library/Homebrew/language/node.rb

```diff
@@ -1,10 +1,44 @@
+require "json"
+
 module Language
   module Node
     def self.npm_cache_config
       "cache=#{HOMEBREW_CACHE}/npm_cache"
     end
 
-    def self.pack_for_installation
+    def self.node_system(cmd, *args)
```

Comment:
Would be good to use the formula system method instead of reimplementing it here.